newbie_guide.md

```diff
@@ -0,0 +1,211 @@
+# A complete beginner guide to BTC Anchoring
+
+This manual is intended for people who are about to use `exonum-btc-anchoring` for the first time.
+
+It verbose describes the process of compiling and launching the `btc_anchoring` example within
+`run-dev` mode on the linux.
+
+## Step 1. Preparing `bitcoind`
+
+- Download `bitcoind`: [https://bitcoin.org/en/download](https://bitcoin.org/en/download)
+- Install `bitcoind`:
+
+    ```sh
+    tar xzf bitcoin-0.18.0-x86_64-linux-gnu.tar.gz
+    sudo install -m 0755 -o root -g root -t /usr/local/bin bitcoin-0.18.0/bin/*
+    ```
+
+- Create a bitcoin config file: `~/.bitcoin/bitcoin.conf` with the following content:
+
+    ```sh
+    server=1
+    regtest=1
+    txindex=1
+    rpcuser=user
+    rpcpassword=password
+    ```
+
+- Run the `bitcoind` in the daemon mode:
+
+    ```sh
+    bitcoind --daemon
+    ```
+
+- Now `bitcoind` is running in the `regtest` mode. We should mine some blocks to obtain balance on
+our account:
+
+    Get the address of our wallet:
+
+    ```sh
+    bitcoin-cli -regtest getnewaddress
+    ```
+
+    Mine some blocks (change address to address you've obtained from the previous command):
+
+    ```sh
+    bitcoin-cli -regtest generatetoaddress 100 2NFNp5RbTyEwV8yijYg9sUCHsVApiqov8DA
+    ```
+
+    Verify that balance is now non-zero:
+
+    ```sh
+    bitcoin-cli -regtest getbalance
+    ```
+
+    If balance is still zero, generate 100 more blocks.
+
+## Step 2. Compile & initial run
+
+- Be sure to have `exonum_launcher` installed and updated to the point at which it supports plugins.
```

Comment:
Mmm, isn't it `exonum-launcher`? I'd add a link to the repo.

Reply:
Module name is `exonum_launcher`. Repository name is `exonum-launcher`. We're talking about module, so it's an underscore.